src/unity/python/turicreate/test/test_json.py

```diff
@@ -251,8 +252,36 @@ def test_variant_to_json(self):
         ]]
 
     def test_malformed_json(self):
-        self.assertRaises(IOError, SArray.read_json, './malformed.json')
```

Comment:
Do you want to also delete `malformed.json`?

Reply:
Or keep it, and just switch out `IOError` for `RuntimeError`?

Reply:
Yeah, in a standalone file it is much easier to see why it's malformed.

Reply:
The problem is that there is actually an additional hidden error -- if the test is not run from the current directory, then it can't find the file and raises an IOError.  If it is run from the current directory, it finds it and raises a RuntimeError.  Thus the current test wasn't correct -- in many cases, the actual error raised was the file-not-found.  Writing out the file explicitly solves this.